### DISCUSSION TITLE
AutoGTP - Template

### DISCUSSION THREAD

**Discussion by jlph79** [FlowiseAI/Flowise#185] [Q&A]
Hallo 
I am trying out the autogtp template, and have configured all the components. But I am getting the following error in the chat:
TypeError: Cannot read properties of undefined (reading 'length')

Any ideas on how to debug this?
thanks in advance
gr
Jose

**Comment by SongLiangZheng** [comment]
+1

**Reply by SongLiangZheng** [reply]
..
